### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by mayank99**
- Added FAQ link in README.md
- Added visual testing information in CONTRIBUTING.md
- Added visual testing checklist item in PR template
- Added FAQ link in bug report template
- Added tippy.js link in JSDoc comments in a few places.
- Updated external `@link` tags to use markdown syntax.

### CODE REVIEW

File: src/core/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -22,7 +22,8 @@ export type TooltipProps = {
   Omit<CommonProps, 'title'>;
 
 /**
- * Basic tooltip component
+ * Basic tooltip component to display informative content when an element is hovered or focused.
+ * Uses the Popover component, which is a wrapper around {@link https://atomiks.github.io/tippyjs tippy.js}.
```
Comment (mayank99) [inline src/core/Tooltip/Tooltip.tsx:26]: Damnit, VSCode 1.57 broke this nice feature
image

Comment (mayank99) [reply src/core/Tooltip/Tooltip.tsx:26]: I see it's fixed in https://github.com/microsoft/vscode/commit/7339e948305024dbf0479e926e7e5bccb8baa97b but I think that might take a while to release.

Meanwhile I've replaced those tags with markdown syntax.

**Review by bentleyvk — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ yarn add @itwin/itwinui-react
 
 ---
 
-## Use
+## Usage
```
Comment (veekeys) [inline README.md:33]: if you change here, then change install -> installation too..
Why use is wrong?

**Review by veekeys — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mayank99) [reply README.md:33]: Idk, I've never seen "Use" in popular libraries. I've mostly seen "Usage" and "Examples".

But I've seen both "Install" and "Installation".

**Pull request merged**